modules/mapping-utils/src/createConnectionTypeDefs.js

```diff
@@ -28,6 +28,7 @@ export default ({ type, fields = '', createStateTypeDefs = true }) => `
       first: Int
       last: Int
       searchAfter: JSON
+      trackTotalHits: Boolean
```

Comment:
You can provide the default value here in the schema actually, this is actually preferred so the API consumer can see it :)

Reply:
added in https://github.com/overture-stack/arranger/pull/627/commits/5774e409d0fe578469e37798268b270b4bc9621b !